database/src/main/01_users.ddl

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 ABSA Group Limited
```

Comment:
```suggestion
 * Copyright 2021 ABSA Group Limited
```

Reply:
Yup. Copyrights need fixing and adding where missing.